index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Daniel's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+<body class="body-container">
+    <header class="header">
+        <span class="navitem name">Daniel Malo</span>
+        <a class="button navitem button hamburger">
+            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 .5 16 16" fill="currentColor" class="bi bi-list">
+                <path fill-rule="evenodd" d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"/>
+            </svg>
+        </a>
+    </header>
+    <div class="content-container content decorative-background">
+        <h1>
+            <span>Hey There.</span>
+            <span>I'm Daniel</span>
+        </h1>
+        <section>
+            <h3>I'm a Software Developer</h3>
+            <div class="self-intro">
+                I can help you build a product, feature or website. Look through some of 
+                my work and experience it! If you like what you see and have a project 
+                you need coded, please don't hesitate to contact me.
+            </div>
+        </section>
+        <div class="media-icons">
+            <img class="media-icon" src="Icons/Vector.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-1.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-2.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-3.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-4.png" alt="no image available">
+        </div>
+    </div>
```

Comment:
- [ ]  Nice work here, it would be better if you used a `section` tag instead of a `div` for this section.